### ISSUE TITLE
training stop

### ISSUE THREAD

**Issue by YoshikawaMasashi** [tensorflow/magenta#59]
Training loop stop.

I set num_training_steps=20000 but Training stop when 10 steps.

OS is Ubuntu 64bit. and tensorflow is without GPU.

When this issue occured, I used ten eagles midi file of this site(http://www.midiworld.com/files/142/).
The_Eagles_-_Best_of_My_Love.mid,The_Eagles_-_Desperado.mid.The_Eagles_-_Heartache_Tonight.mid,The_Eagles_-_One_of_These_Nights.mid,The_Eagles_-_Pretty_Maids_All_In_A_Row.mid,The_Eagles_-_Take_It_Easy.mid,The_Eagles_-_Take_It_To_The_Limit.mid,The_Eagles_-_The_Last_Resort.mid,The_Eagles_-_Try_And_Love_Again.mid,The_Eagles_-_Wasted_Time.mid

terminal 

---------------------------------------------------------
Starting training loop
Wrote checkpoint to /tmp/basic_rnn/run1/train/basic_rnn.ckpt-10
Global Step: 10 - Loss: 290226.031 - Log-perplexity: 1.277 - Step Accuracy: 0.79 - Avg Accuracy (last 20 summaries): 0.74 - Learning Rate: 0.500000
(tensorflow)marshi@ubuntu:~/Desktop/magenta

---------------------------------------------------------

Thank you

**Comment by michaelcaterisano** [comment]
I think you just have to let it keep running and it will continue through the rest of the steps.

**Comment by YoshikawaMasashi** [comment]
I tried to do this, but rnn training don't keep running.

I don't always get this issue.

When I used a certain midi file, I get this error.
And when I set 'rnn_raler_sizes' high (example:100), I get this error.

**Comment by cinjon** [comment]
@YoshikawaMasashi, this is pretty hard for us to diagnose. Have you had any further success?

**Comment by cghawthorne** [comment]
Closing due to lack of updates.

**Comment by buaatian** [comment]
hey, I meet the same problem like yours. However, every time I tried to run my code, it always had this wrong massage:

_INFO:tensorflow:hparams = {'rnn_layer_sizes': [128, 128], 'attn_length': 40, 'dropout_keep_prob': 0.5, 'batch_size': 128, 'decay_rate': 0.97, 'clip_norm': 3, 'initial_learning_rate': 0.001, 'decay_steps': 1000, 'skip_first_n_losses': 0}
INFO:tensorflow:Train dir: /tmp/melody_rnn/logdir/run1/train
INFO:tensorflow:Starting training loop...
INFO:tensorflow:global_step/sec: 0
**trainthemodel.sh: line 6: 13536 Killed**                  melody_rnn_train --config=attention_rnn --run_dir=/tmp/melody_rnn/logdir/run1 --sequence_example_file=/tmp/melody_rnn/sequence_examples/training_melodies.tfrecord --hparams="{'batch_size':64,'rnn_layer_sizes':[64,64]}" --num_training_steps=20000_

I think this isn't correct. Does anyone know how to solve it?
